src/main/java/org/openhmis/manager/ExitManager.java

```diff
@@ -298,4 +297,4 @@ public static TmpExit generateTmpExit(ExitDTO inputDTO) {
 		return tmpExit;
 	}
 	
```

Comment:
Ah, you're saying that these two methods on class ExitManager should be removed because no one uses them anymore:

    public static List<ExitDTO> getExits(Date updateDate)
    public static ExitDTO getExitByEnrollmentId(String enrollmentId)

?  Can do; might do after the merge instead of before, unless you have a strong opinion in favor of doing it before.